anyio/_backends/_trio.py

```diff
@@ -2,11 +2,11 @@
 from types import TracebackType
 from typing import Callable, Optional, List, Type, Union
 
-import trio.hazmat
+import trio.lowlevel
```

Comment:
This module does not exist in trio before v0.15. It would be better to use a conditional import for now.

Reply:
Done. (For some values of "done" …)